docs/REST API/rest-api.md

```diff
@@ -6989,6 +6989,7 @@ Deletes the session specified by ID. When the user associated with the session n
         "name": "glibc",
         "version": "2.12",
         "source": "rpm_packages",
+        "extension_id": "",
```

Comment:
Generally we don't return an empty string -- it would be omitted from response.

Reply:
@getvictor in that case, we should be consistent with how we're already doing things in this endpoint for now and leave it out if it's not there. I'll bring a request to the next Feature Fest to handle this in a way that's more consistent with all the other API endpoints (aka response that includes all the keys, even if that means some empty values). I'd be curious to find out if there was an intentional reason for having this specific endpoint behave differently.

Reply:
Many endpoints omit empty strings from the response. In the code, this is done with `omitempty` annotation like:

```
server/fleet/scheduled_queries.go:	QueryName   string `json:"query_name,omitempty" db:"query_name"`
```

This is best practice for performance. When we need to return 1000s of records, it is faster to transmit less data.

Reply:
Got it. I wonder if some of our REST API docs are out-of-date if `omitempty` is being used in most responses (there's a lot of "" in those examples)